assets/src/editor/blocks/donation-bar/index.css

```diff
@@ -0,0 +1,117 @@
+.wp-block-shiro-donation-portal-form form {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    gap: var(--wp--preset--spacing--50);
+}
+
+.wp-block-shiro-donation-portal-form fieldset {
+    border: 0;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    gap: var(--wp--preset--spacing--40);
+}
+
+.wp-block-shiro-donation-portal-form legend {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    visibility: hidden;
```

Comment:
Could we have piggybacked on a global .screen-reader-text here...?

Reply:
Sara showed us some future designs on the standup yesterday where these would be displayed, so I figured we can add a block style for those when the time comes.